src/AesEncryptingStream.php

```diff
@@ -105,9 +105,7 @@ private function encryptBlock(int $length): string
         } while (strlen($plainText) < $length && !$this->stream->eof());
 
         $options = OPENSSL_RAW_DATA;
-        if (!$this->stream->eof()
-            || $this->stream->getSize() !== $this->stream->tell()
-        ) {
+        if (!$this->stream->eof()) {
```

Comment:
Could encryption potentially have the same issue? Might we need to make sure we have read all the data and apply the `|= OPENSSL_ZERO_PADDING` accordingly?

Reply:
I don't believe the same issue would be encountered when encrypting. To be on the safe side, `AesEncryptingStream` is now tested with the same scenario as it was in #11 (i.e., [reading beyond the end of the stream](https://github.com/jeskew/php-encrypted-streams/pull/13/files#diff-a3a84da543c2d8d07bc0b9383905bbe1R53)), but no code changes were required to get the test cases to pass.